src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/PackageDeclarationCheckTest.java

```diff
@@ -69,6 +70,29 @@ public void testOnFileWithCommentOnly() throws Exception {
     @Test
     public void testCorrectFile() throws Exception {
```

Comment:
You should use appropriate name for the test. Think what it does, what its purpose is.